app/controllers/tweet_controller.rb

```diff
@@ -0,0 +1,87 @@
+class TweetsController < ApplicationController
+
+
+	get '/tweets' do
+		if logged_in?
+			@user = current_user
+			success
+			erb :'/tweets/tweets'
+		else
+			please_login
+			redirect '/login'
+		end
+	end
+
+
+	get '/tweets/new' do
+		if logged_in?
+			@user = current_user
+			erb :'/tweets/create_tweet'
+		else
+			please_login
+			redirect '/login'
+		end
+	end
+
+	post '/tweets/new' do
+		if filled_out(params)
+			@tweet = Tweet.create(content: params[:content])
+			@tweet.id = Tweet.all.last.id
```

Comment:
Primary keys are set automatically on persistence -- you won't want to set them manually.